README.md

```diff
@@ -28,9 +28,14 @@ jobs:
       - name: Moropo - Trigger Mobile App Test Run
         uses: moropo-com/action-trigger-test-run@v1.0.0
         with:
-          scheduled_test_id: ...
           app_secret: ${{ secrets.MOROPO_APP_SECRET }}
-          expo_release_channel: ...
+          
+          # REPLACE WITH YOUR OWN
+          scheduled_test_id: 3a8c2d7b-9e0f-4b2c-a7d4-6b8f7a9c5e10
+          
+          # REPLACE WITH YOUR OWN
+          expo_release_channel: https://u.expo.dev/5b2c7f8a-9d34-4c1e-8a5f-3d9e7b0c2f12?channel-name=moropo-410&runtime-version=exposdk:47.0.0&platform=android&disableOnboarding=1
```

Comment:
Should this param be suffixed `_url` to make it more distinct? What does Expo call it?

Reply:
They call it Expo Release Channel and we do call it the same in the platform